scripts/man.py

```diff
@@ -136,7 +136,7 @@ def format_bi(op):
         else:
             defval = op['default_val']
         print('\\fR  Default Value: {} \\fR'.format(defval))
-    print()
+    print('')
```

Comment:
It think it's a python 2.7 vs python 3 problem... in python 3 `print()` should work – which is what I wanted. But if you run it with python 2.7 then it prints `()`. I guess I could also add 

```
from __future__ import print_function
```

to the top of the file and have a more consistent behavior... 🤔  I'll do that 😄